src/components/tabs/sell.tab.tsx

```diff
@@ -24,6 +18,15 @@ import {
   StyledTabProps,
   StyledVerticalStack,
 } from '@dfx.swiss/react-components';
+import {
+  Asset,
+  Blockchain,
+  useAssetContext,
+  useAuthContext,
+  useSessionContext,
+  useUserContext,
+} from '@dfx.swiss/react';
+import { AssetType } from '@dfx.swiss/react/dist/definitions/asset';
```

Comment:
Why is `AssetType` import separated from the rest? 🤔

Reply:
good point. because it was not correctly exporting in the package.